### ISSUE TITLE
setting std.lv=TRUE does not set auto.fix.first=FALSE

### ISSUE THREAD

**Issue by TDJorgensen** [yrosseel/lavaan#154]
I'm sure this is due to adjusted logic somewhere after adding the new `effect.coding=` argument.  Hopefully this is straight-forward to fix.

```
example(cfa)
update(fit, std.lv=TRUE)@Options[c("std.lv","auto.fix.first")] # both true
```

**Comment by TDJorgensen** [comment]
Reminder to post when this is resolved:

https://groups.google.com/d/msg/lavaan/U17jLbD95Ls/aXuroLTXAwAJ

**Comment by yrosseel** [comment]
Note that 

fit <- sem(HS.model, data = HolzingerSwineford1939, std.lv = TRUE)
fit@Options[c("std.lv","auto.fix.first")] # true and false!

The reason is in the sem/cfa/growth functions, we set
mc$auto.fix.first  = !std.lv

In other words, this is (currently) handled outside lav_options_set(), and therefore not taken care of in the update() function. I will move this 'logic' to lav_options_set().

**Comment by yrosseel** [comment]
(Was already fixed in 0.6-5; forgot to close)
